appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleEvent.kt

```diff
@@ -92,6 +95,14 @@ internal sealed class ExperienceLifecycleEvent(
             "localeName" to experience.localeName,
             "localeId" to experience.localeId,
         ).apply {
+            // additional properties based experience trigger
+            when (val trigger = experience.trigger) {
+                is ExperienceCompletionAction -> this["from_experience_id"] = trigger.fromExperienceId
+                is LaunchExperienceAction -> this["from_experience_id"] = trigger.fromExperienceId
+                is PushNotification -> this["push_notification_id"] = trigger.fromPushId
+                else -> Unit
+            }
```

Comment:
lets do camelCase prop names here to match ios ([per slack chat](https://appcues.slack.com/archives/C031RLGS4F8/p1720795696077369?thread_ts=1719942505.595759&cid=C031RLGS4F8))